ArchiMetrics.Analysis/Metrics/NamespaceMetricsCalculator.cs

```diff
@@ -29,7 +29,8 @@ public NamespaceMetricsCalculator(SemanticModel semanticModel)
 		public INamespaceMetric CalculateFrom(NamespaceDeclarationSyntaxInfo namespaceNode, IEnumerable<ITypeMetric> metrics)
 		{
 			var typeMetrics = metrics.AsArray();
-			var linesOfCode = typeMetrics.Sum(x => x.LinesOfCode);
+			var linesOfCode = typeMetrics.Sum(x => x.LinesOfCode);
+            var sourceLinesOfCode = typeMetrics.Sum(x => x.SourceLinesOfCode);
```

Comment:
Similar to error in type metrics calculation, this calculation does not include namespace declaration lines. Furthermore, using statements may happen outside of the namespace declaration, where do they get counted? In project metrics?